example-apps/workplace-search/api/llm_integrations.py

```diff
@@ -0,0 +1,45 @@
+from langchain.chat_models import ChatOpenAI, ChatVertexAI, AzureChatOpenAI, BedrockChat
+import os
+import google.auth
+import boto3
+
+LLM_TYPE = os.getenv("LLM_TYPE", "openai")
+
+def init_openai_chat():
+    OPENAI_API_KEY = os.getenv("OPENAI_API_KEY")
+    return ChatOpenAI(openai_api_key=OPENAI_API_KEY, streaming=True, temperature=0.2)
+def init_vertex_chat():
+    credentials, project = google.auth.default(project_id="vertex-ai-309406")
```

Comment:
im assuming that your vertex support PR will improve passing in the project_id as a config variable?